src/View/Form/ContextInterface.php

```diff
@@ -44,6 +44,15 @@ public function isPrimaryKey($field);
     public function isCreate();
 
     /**
+     * Sets the type of the form the context is created from.
+     *
+     * @param string $requestType The request type of the form the
+     *   context is created from.
+     * @return void
+     */
+    public function setRequestType($requestType);
```

Comment:
Is this method/parameter really necessary? Seems like adding a constructor parameter would work just as well.